rollup.config.js

```diff
@@ -0,0 +1,22 @@
+import babel from 'rollup-plugin-babel';
+import pkg from './package.json'
+
+export default {
+  input: 'src/index.js',
+  output: [
+    {
+      file: pkg.main,
+      format: 'cjs',
+    },
+    {
+      file: pkg.module,
+      format: 'esm',
+    }
+  ],
+  plugins: [
+    babel({
+      babelrc: false,
```

Comment:
That's unfortunately not easily possible. jest and rollup need a slightly different config, rollup's needs to preserve the modules as ESMs while jest needs them transformed to CommonJS.

It might be possible to get jest working with ESM using something like @std/esm but this seems much easier.

Reply:
Ah, I see. Thanks for the explanation!